src/main/java/de/muenchen/isi/infrastructure/entity/stammdaten/Umlegung.java

```diff
@@ -0,0 +1,33 @@
+package de.muenchen.isi.infrastructure.entity.stammdaten;
+
+import de.muenchen.isi.infrastructure.entity.BaseEntity;
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Entity
+@Data
+@EqualsAndHashCode
+public class Umlegung extends BaseEntity {
+
+    @Column(nullable = false)
+    private String bezeichnung;
+
+    @Column(nullable = false)
+    private LocalDate gueltigAb;
```

Comment:
Da die selbe Förderart (Attribut `bezeichnung`) Umrechnungsschlüssel mit unterschiedlichen Gültigkeitszeiträumen besitzen kann (Attribut `gueltigAb`), ist für die beiden Attribute `bezeichnung` und `gueltigAb` ein kombinierter Fachlicher Tabellenschlüssel zu definieren. Ansonsten ist es in der Tabelle möglich für einen Förderart mehrere Umlegungsentitäten mit dem selben `gueltigAb`-Datum anzulegen.

Reply:
siehe z.B. UniqueConstraint bei der Abfragevariante